app/lib/travel_planner/journey.rb

```diff
@@ -24,7 +24,12 @@ def format_legs(journey_data)
     journey = journey_data['Leg'].map do |leg_data|
       leg = TravelPlanner::Leg.new leg_data, @coords
 
-      formatted_leg = leg.type=='BIKE' ? format_bike(leg) : format_public(leg)
+      formatted_leg = case leg.type
+      when 'BIKE', 'WALK'
+        format_bike(leg)
```

Comment:
what happens if you if:
you can't bike? (i assume you then push the bike)
you can't even push a bike?